showkase-processor-testing/src/androidTest/java/com/airbnb/android/showkase_processor_testing/ShowkaseBrowserTestFlows.kt

```diff
@@ -45,10 +47,27 @@ internal fun AndroidComposeTestRule<ActivityScenarioRule<ShowkaseBrowserActivity
     vararg textList: String
 ) {
     textList.forEach {
-        onNodeWithText(it).assertDoesNotExist()
+        onNodeWithText(it).assertEitherDoesNotExistOrIsNotDisplayed()
     }
 }
 
+/**
+ * LazyColumn pre-fetching may precompose items in advance even if they are not displayed. There is
+ * not yet a built-in assertion to determine whether a node is not visible for any reason, as
+ * assertIsNotDisplayed() fails if the node does not exist.
+ * See https://issuetracker.google.com/issues/187188981
+ */
+private fun SemanticsNodeInteraction.assertEitherDoesNotExistOrIsNotDisplayed() {
```

Comment:
is this related to a change introduced in beta08 or did I miss this from a previous release. Do you know the CL for this? Might be useful to add a link to that here.

Reply:
Seems it's actually since beta07 - and I'm guessing the failures I was seeing were probably actually caused by the "LazyColumn/Row will now keep up to 2 previously visible items active (not disposed)" part - https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.0.0-beta07

Tweaked the comment to point to that CL

Reply:
Thank you kind sir!